indexservice/indexer.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/iotexproject/iotex-core/action"
 	"github.com/iotexproject/iotex-core/blockchain/block"
 	"github.com/iotexproject/iotex-core/config"
-	"github.com/iotexproject/iotex-core/db/rds"
+	s "github.com/iotexproject/iotex-core/db/sql"
```

Comment:
no need to have alias

Reply:
this conflict with database/sql, I use alias to differentiate them